base_location_geonames_import/wizard/geonames_import.py

```diff
@@ -48,135 +47,77 @@ def transform_city_name(self, city, country):
         :param country: Country record
         :return: Transformed city name
         """
-        return city
+        res = city
```

Comment:
As char variables are passed as value, not as reference, you can modify directly over existing one (`city = city.title()`), but that's only a technical nitpicking for saving only line and variable creation.